.github/workflows/release.yaml

```diff
@@ -18,8 +18,7 @@ jobs:
           path: source
           fetch-depth: 0
       - id: detect-profiles
-        working-directory: source
```

Comment:
not sure you can change this

Reply:
it might break the build

Reply:
`.github/actions/detect-profiles` doesn't accept `working-directory`.
[Prepare Run Failed](https://github.com/emqx/emqx/actions/runs/3265399016/workflow#L34)
[Prepare Run Succeeded](https://github.com/emqx/emqx/actions/runs/3280141977/workflow#L34)